nodejs/eks/securitygroup.ts

```diff
@@ -47,9 +47,9 @@ export function createNodeGroupSecurityGroup(name: string, args: NodeGroupSecuri
             args.tags,
             args.eksCluster.name,
         ]).apply(([tags, clusterName]) => (<aws.Tags>{
+            ...tags,
             "Name": `${name}-nodeSecurityGroup`,
             [`kubernetes.io/cluster/${clusterName}`]: "owned",
-            ...tags,
```

Comment:
Curious why you made this change?  It seems more correct to allow users to override these defaults if the really want to?

Reply:
> Any reason not to apply ...tags and ...asgTags last?

For the quoted and this comment - you're right. It makes sense to let users override and put `tags` last. I'll make the change, thx!